aten/src/ATen/native/quantized/cpu/qnnpack_utils.h

```diff
@@ -10,15 +10,17 @@ struct QnnpackOperatorDeleter {
   }
 };
 
-// PackedWeight struct stores the original Weight and Bias as QNNPACK currently
-// does not support an unpack function.
-// Possible optimiation - For PyTorch Mobile, once the model is scripted and
-// serialized we don't need to call unpack, so we can save some memory by
-// checking for this case.
+// PackedWeight struct for QNNPACK stores the original Weight and Bias as
+// QNNPACK currently does not support an unpack function. Possible optimiation -
+// For PyTorch Mobile, once the model is scripted and serialized we don't need
+// to call unpack, so we can save some memory by checking for this case. Input
+// scale is set to 0 in pre-pack step. During runtime if input scale value
```

Comment:
I'd add more explanation here on:
- why you need it: qnnpack uses bias quantized with input scale
- why it's hacky but ok: we want to fix it properly, but mutability of the matrix is ok as long as the graph is static with the same input scale coming in all the time